<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="3" >
     <widget class="QLineEdit" name="equationLineEdit" />
    </item>
    <item row="0" column="3" >
     <widget class="QLabel" name="label_3" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Independent Variable:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" >
     <widget class="QLineEdit" name="independentVariableLineEdit" >
      <property name="maximumSize" >
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="5" >
     <widget class="QLabel" name="label_2" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Range:</string>
      </property>
      <property name="textFormat" >
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="6" >
     <widget class="QDoubleSpinBox" name="lowerDoubleSpinBox" >
      <property name="minimum" >
       <double>-99.989999999999995</double>
      </property>
     </widget>
    </item>
    <item row="0" column="7" >
     <widget class="QDoubleSpinBox" name="higherDoubleSpinBox" >
      <property name="minimum" >
       <double>-99.989999999999995</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Constants:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item rowspan="4" row="1" column="1" colspan="2" >
     <widget class="QTableView" name="constantsTableView" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize" >
       <size>
        <width>256</width>
        <height>100</height>
       </size>
      </property>
      <property name="editTriggers" >
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="tabKeyNavigation" >
       <bool>true</bool>
      </property>
      <property name="showDropIndicator" stdset="0" >
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode" >
       <bool>false</bool>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
     </widget>
    </item>
    <item rowspan="4" row="1" column="3" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>97</height>
       </size>
      </property>
     </spacer>
    </item>
    <item rowspan="2" row="1" column="5" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>72</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" >
     <widget class="QPushButton" name="addConstantsModelPointPushButton" >
      <property name="text" >
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QPushButton" name="clearConstantsModelPushButton" >
      <property name="text" >
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="3" >
     <widget class="QPushButton" name="evaluatePushButton" >
      <property name="text" >
       <string>Evaluate</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>48</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="5" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>72</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="8" >
     <widget class="QLabel" name="equationLabel" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Equation</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2" >
     <widget class="QTextBrowser" name="textBrowser" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="6" >
     <widget class="GraphWidget" native="1" name="graphWidget" >
      <property name="acceptDrops" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header>GraphWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
